### PR TITLE
Scalardl tutorial updation with scalardl-java-client-sdk 2.0.4

### PR DESCRIPTION
**Pull request by scalar-boney**
Sample application and documentation updated with scalardl-java-client-sdk 2.0.4

### CODE REVIEW

**Comment by feeblefakie** [comment]
@scalar-boney Can you add me as a reviewer ?

File: README.md [hunk 1]
```diff
@@ -319,45 +317,44 @@ for (Path contract : contractList) {
...
 
 ```java
-ContractExecutionResponse response = clientService.executeContract(contractId, argument);
-if (response.getStatus() != StatusCode.OK.get()) {
-  System.err.println("Error during contract execution");
-  System.err.println("Status code: " + response.getStatus());
-  System.err.println("Message: " + response.getMessage());
-  return;
+try {
+   ContractExecutionResult result = clientService.executeContract(contractId, object);
```
Comment (feeblefakie) [inline README.md]: I think indent is wrong. Use 2 please.

File: manual/README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ java -jar am-1.0.jar list [-h] asset_type
 To add a new asset to the specified category.
 
 ```
-java -jar am-1.0.jar [-h] asset_type asset
+java -jar am-1.0.jar add [-h] asset_type asset
```
Comment (feeblefakie) [inline manual/README.md]: 👍

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java [hunk 3]
```diff
@@ -50,15 +49,15 @@ protected void executeContract(String contractName, JsonObject contractParameter
                 (contractParameter != null)
                     ? contractParameter
                     : Json.createObjectBuilder().build();
-            ContractExecutionResponse response = clientService.executeContract(contractId, object);
-            if (response.getStatus() != StatusCode.OK.get()) {
+            try {
+              ContractExecutionResult result = clientService.executeContract(contractId, object);
+              if(result.getResult().isPresent()) {
+                prettyPrintJson(result.getResult().get());
```
Comment (feeblefakie) [inline src/main/java/com/scalar/am/command/LedgerClientExecutor.java]: I think `prettyPrintJson` accepts `null` so we can remove the if statement.

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java [hunk 4]
```diff
@@ -72,11 +71,10 @@ protected void validateAsset(String id) {
     try {
       LedgerExecutorFunction f =
           (clientService) -> {
-            LedgerValidationResponse response = clientService.validateLedger(id);
-            if (response.getStatus() != StatusCode.OK.get()) {
+            LedgerValidationResult result = clientService.validateLedger(id);
+            if (result.getCode().get() != StatusCode.OK.get()) {
```
Comment (feeblefakie) [inline src/main/java/com/scalar/am/command/LedgerClientExecutor.java]:
```suggestion
            if (result.getCode().equalsTo(StatusCode.OK) {
```

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java [hunk 5]
```diff
@@ -104,7 +102,6 @@ protected final void executeOnLedger(LedgerExecutorFunction f) throws Exception
   /**
    * Pretty print the json to the standard output
    *
-   * @param json the json to print
```
Comment (feeblefakie) [inline src/main/java/com/scalar/am/command/LedgerClientExecutor.java]: Why it is deleted ?

File: README.md [hunk 6]
```diff
@@ -319,45 +317,44 @@ for (Path contract : contractList) {
...
 It is to be noted that any registered user can validate any asset of the ledger.
 Check [ValidateAsset](./src/main/java/com/scalar/am/command/ValidateAsset.java).
 
 ```java
-LedgerValidationResponse response = clientService.validateLedger(id);
-if (response.getStatus() != StatusCode.OK.get()) {
-  System.err.println(response.getMessage());
-  return;
+LedgerValidationResult result = clientService.validateLedger(id);
+if (result.getCode().get() != StatusCode.OK.get()) {
```
Comment (feeblefakie) [inline README.md]:
```suggestion
if (result.getCode().equalsTo(StatusCode.OK.get()) {
```

File: README.md (hunk 1 above)
Comment (scalar-boney) [reply README.md]: Thank you for review comment.
Indent corrected.

File: README.md [hunk 7]
```diff
@@ -265,37 +265,35 @@ Notice that we need a ClientConfig to create a ClientService.
...
-scalar.ledger.client.server_host=<address of one server of the Scalar DL network>
-scalar.ledger.client.server_port=<port of one server of the Scalar DL network>
-scalar.ledger.client.cert_holder_id=<holder id>
-scalar.ledger.client.cert_path=<path to certificate>
-scalar.ledger.client.cert_version=<version of certificate>
-scalar.ledger.client.private_key_path=<path to private key>
-scalar.ledger.client.tls.enabled=<true to enable TLS between Scalar DL network>
-scalar.ledger.client.authorization.credential=Basic: <credential>
+scalar.dl.client.server.host=<address of one server of the Scalar DL network>
+scalar.dl.client.server.port=<port of one server of the Scalar DL network>
```
[the remark this answers is not in the document]
Comment (scalar-boney) [reply README.md]: Thank you for review comment.
I have checked in the scalar repository but couldn't get the exact one.
```
scalar.dl.ledger.server.admin_port=<admin port of one server of the Scalar DL network>
```
Do you mean the above config?
Could you please confirm, I will integrate.

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java (hunk 4 above)
Comment (scalar-boney) [reply src/main/java/com/scalar/am/command/LedgerClientExecutor.java]: Thank you for review comment.
Changes updated in f17934f22814fb2c17e217e4bc5f8932f9dfaead

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java (hunk 5 above)
Comment (scalar-boney) [reply src/main/java/com/scalar/am/command/LedgerClientExecutor.java]: Sorry, removed by mistake. 
Now it is added

File: README.md (hunk 6 above)
Comment (scalar-boney) [reply README.md]: Thank you for review comment.
Changes updated in f17934f

File: src/main/java/com/scalar/am/command/LedgerClientExecutor.java (hunk 3 above)
Comment (scalar-boney) [reply src/main/java/com/scalar/am/command/LedgerClientExecutor.java]: Thank you for review comment.
Changes updated in 6822d7ba27e9704795d16438fa2c57d1beb03c24

**Pull request closed without merging**